geode-core/src/main/java/org/apache/geode/internal/net/BufferPool.java

```diff
@@ -69,7 +74,8 @@ public BufferPool(DMStats stats) {
   /**
    * use direct ByteBuffers instead of heap ByteBuffers for NIO operations
    */
-  public static final boolean useDirectBuffers = !Boolean.getBoolean("p2p.nodirectBuffers");
+  public static final boolean useDirectBuffers = !(Boolean.getBoolean("p2p.nodirectBuffers")
+      || Boolean.getBoolean(GeodeGlossary.GEMFIRE_PREFIX + "noDirectBuffers"));
```

Comment:
is the GF prefixed version of the property need to be exposed/documented?

Reply:
> is the GF prefixed version of the property need to be exposed/documented?

yes, but I'm not sure what the best way is to do that.  I updated properties.html in this PR with the new property.